sdk/keyvault/azcertificates/constants.go

```diff
@@ -0,0 +1,113 @@
+//go:build go1.16
+// +build go1.16
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azcertificates
+
+// ActionType - The type of the action.
+type ActionType string
+
+const (
+	ActionTypeEmailContacts ActionType = "EmailContacts"
+	ActionTypeAutoRenew     ActionType = "AutoRenew"
+)
```

Comment:
`CertificatePolicyAction`:

https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificatePolicyAction.cs